src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -272,6 +273,29 @@ public void onTraversalStart() {
                 "The SAM sequence dictionary is either not the same for all of the model shards, " +
                         "or is different from the SAM sequence dictionary of calls shards.");
 
+        return sequenceDictionary;
+    }
+
+    /**
+     * Performs various validations on input arguments. Since many of these validations requires loading and parsing
+     * reusable data, we store them as global variables (shard interval lists, sample name, etc).
+     */
+    @Override
+    public void onTraversalStart() {
+        validateArguments();
+
+        numShards = inputUnsortedCallsShardPaths.size();
+
+        /* get intervals from each call and model shard in the provided (potentially arbitrary) order */
+        final List<SimpleIntervalCollection> unsortedIntervalCollectionsFromCalls =
+                getIntervalCollectionsFromPaths(inputUnsortedCallsShardPaths);
+        final List<SimpleIntervalCollection> unsortedIntervalCollectionsFromModels =
+                getIntervalCollectionsFromPaths(inputUnsortedModelShardPaths);
+
+        if (sequenceDictionary == null) {
```

Comment:
I don't think you have to wrap this in a null check because it's also checked in getBestAvailableSequenceDictionary()

Reply:
Just so I understand, are we allowing overriding of the sequence dictionary in the shards (and skipping the consistency check) by allowing the parameter `--sequence-dictionary` to be specified?  If so, we might want to document.  Otherwise, I'd be inclined to enforce using the sequence dictionary in the shards (and ensuring the consistency check across shards is performed) by changing the null check in getBestAvailableSequenceDictionary to a check that the dictionary has *not* been set via the command line.

Reply:
Actually, I'm not sure if that sequence dictionary ever gets hooked up, in which case I guess I'm just confused by the intent of the null check.